### ISSUE TITLE
Profiler sums timings on Task.WhenAll

### ISSUE THREAD

**Issue by junalmeida** [MiniProfiler/dotnet#554]
When running parallel tasks with `Task.WhenAll`, profiler UI is summing all nested timings when it should be displaying like a gantt chart, or using the timing of the last task finished.

**Comment by NickCraver** [comment]
Do you have an example here? Or ideas? Seriously, even MS Paint works. A proposed display would be good to ideate on here.

It's a hard problem because _there isn't one value_. There's the total thread total wall time for all operations, and then there's aggregate linear time across a parented operation. Some people want to see one, some people want to see the other.

**Comment by junalmeida** [comment]
Yeah, so a discussion which way that info should be displayed is good. But I think at least as a start it should not sum as this is simply incorrect (because nested tasks are in parallel). I think you can display nested tasks like it is done today, but simply do not sum timings.
